### DISCUSSION TITLE
How to change auth method for users

### DISCUSSION THREAD

**Discussion by temrix** [requarks/wiki#4008] [Help / Questions] [answered]
Hello!

I have a handful of users on my Wiki and want to change their authentication strategy by user from `Local` to `Discord Login`. I can also tell my users to change it on my Wiki themselves. Is this currently possible and if so, how?

I have already configured the auth method with Discord. I would prefer if it worked without being a member on a specific Discord server but as there is already one, it is by no means a strict requirement.

**Comment by NGPixel** [comment, marked as answer]
This is not possible atm. You would need to create new users for them.

This is being addressed in the upcoming 3.0 release.

**Reply by temrix** [reply]
Thank you so much! As I understand it, version 2 still needs work to be put into and version 3.0 is not yet ready to be considered as near. Is there some issue/milestone I can track for the upcoming release or get notified whenever it will be ready?

**Reply by NGPixel** [reply]
No more work is being put into 2.0 (other than security issues). Follow on [Twitter](https://twitter.com/requarks) or the [blog](https://blog.js.wiki/).
